src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml

```diff
@@ -206,11 +217,6 @@
   tags:
     - pgbouncer
 
-# Allows to excecute task only when a tag is specified: https://serverfault.com/a/748864
-- shell: /bin/true
-  changed_when: false
-  register: no_tags
-
 - name: Restarts pgbouncer (affect max open files limit)
```

Comment:
Interesting. How come this one didn't do it?

Reply:
this only restart pgbouncer, not postgres